### DISCUSSION TITLE
Help with Parse error

### DISCUSSION THREAD

**Discussion by Mark-Giesen** [WeblateOrg/weblate#10079] [Q&A] [answered]
I recieved the error below, today.  We rearranged the YAML files, so we might have done something wrong somewhere, but is there any way to pinpoint where this error occurred? At what key for instance? Is there a log to be found I don't know about as of yet?

<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:w="urn:schemas-microsoft-com:office:word"
xmlns:m="http://schemas.microsoft.com/office/2004/12/omml"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Word.Document>
<meta name=Generator content="Microsoft Word 15">
<meta name=Originator content="Microsoft Word 15">
<link rel=File-List
href="file:///C:/Users/zurav/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<link rel=Edit-Time-Data
href="file:///C:/Users/zurav/AppData/Local/Temp/msohtmlclip1/01/clip_editdata.mso">

<link rel=themeData
href="file:///C:/Users/zurav/AppData/Local/Temp/msohtmlclip1/01/clip_themedata.thmx">
<link rel=colorSchemeMapping
href="file:///C:/Users/zurav/AppData/Local/Temp/msohtmlclip1/01/clip_colorschememapping.xml">

<style>

</style>

</head>

<body lang=NL link=blue vlink="#954F72" style='tab-interval:35.4pt;word-wrap:
break-word'>

<h1><img width=32 height=32 id="_x0000_i1025"
src="cid:email-logo.png@cid.weblate.org">Parse error <o:p></o:p></h1>

<p class=MsoNormal><a
href="https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fhosted.weblate.org%2F&amp;data=05%7C01%7C%7C3748f3209537496f7b3b08dbc271b000%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C638317565758278412%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&amp;sdata=DwGhRvm%2BnOeH%2BNMnDuJYNTDu%2BvT1nE54webc0q359hc%3D&amp;reserved=0"
target="_blank">Hosted Weblate</a> / <a
href="https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fhosted.weblate.org%2Fprojects%2Fhedy%2F&amp;data=05%7C01%7C%7C3748f3209537496f7b3b08dbc271b000%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C638317565758434155%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&amp;sdata=Mdx5apmVxW66oiW360Ku7YutZa98%2FXgwcgO9%2BzOid6U%3D&amp;reserved=0"
target="_blank">Hedy</a> / <a
href="https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fhosted.weblate.org%2Fprojects%2Fhedy%2Fwebpages%2F&amp;data=05%7C01%7C%7C3748f3209537496f7b3b08dbc271b000%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C638317565758434155%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&amp;sdata=TAnt2lddgKnFTJ%2FpP2EevHWbZ975QzBYJKzT5Dx7IqY%3D&amp;reserved=0"
target="_blank">Webpages</a> <o:p></o:p></p>

<h2>Error message<o:p></o:p></h2>

<pre>unsupported operand type(s) for +=: 'NoneType' and 'str'<o:p></o:p></pre>

<h2>Failing file<o:p></o:p></h2>

<p>content/pages/en.yaml <o:p></o:p></p>

<h2>Component Information <o:p></o:p></h2>

Languages | 47 |  
-- | -- | --
All strings | 20,586 |  
Translated strings | 18,659 | 90%
Untranslated strings | 0 | 0%
Unfinished strings | 1,927 | 9%
Strings marked for edit | 1,927 | 9%

<p class=MsoNormal><a
href="https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fhosted.weblate.org%2Fprojects%2Fhedy%2Fwebpages%2F&amp;data=05%7C01%7C%7C3748f3209537496f7b3b08dbc271b000%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C638317565758434155%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&amp;sdata=TAnt2lddgKnFTJ%2FpP2EevHWbZ975QzBYJKzT5Dx7IqY%3D&amp;reserved=0"
target="_blank">View</a> <o:p></o:p></p>

<p><img border=0 width=32 height=32 id="_x0000_i1026"
src="cid:email-logo-footer.png@cid.weblate.org"><br>
<a
href="https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fweblate.org%2F&amp;data=05%7C01%7C%7C3748f3209537496f7b3b08dbc271b000%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C638317565758434155%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&amp;sdata=r6ZiXlu%2FW1ZqI9RbRFQLkhiHjyv2MgLdznkb6h4VYkw%3D&amp;reserved=0"
target="_blank">Weblate, the libre continuous localization system.</a> <o:p></o:p></p>

<p><span style='font-family:"Cambria Math",serif;mso-bidi-font-family:"Cambria Math"'>▸</span>
<a
href="https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fhosted.weblate.org%2Faccounts%2Fprofile%2F%23notifications&amp;data=05%7C01%7C%7C3748f3209537496f7b3b08dbc271b000%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C638317565758434155%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&amp;sdata=gXlYF35HdVJo1hiuwPI4OOAe37iKvrm8bzhzUf%2BlGiA%3D&amp;reserved=0"
target="_blank">Notification settings</a> <o:p></o:p></p>

<p><span style='font-family:"Cambria Math",serif;mso-bidi-font-family:"Cambria Math"'>▸</span>
<a
href="https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fhosted.weblate.org%2Faccounts%2Funsubscribe%2F%3Fi%3D373778%3A1qmtSf%3A8j2_qYvzb9v2F-vBdN3HYbmxJNMAt6qTy0ruYnpiL0w&amp;data=05%7C01%7C%7C3748f3209537496f7b3b08dbc271b000%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C638317565758434155%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&amp;sdata=TZkUaZ9XxfIVZCQJ3u94Difa84WKQns84%2F1aetX8msI%3D&amp;reserved=0"
target="_blank">Turn off this notification</a> <o:p></o:p></p>

<p class=m8748612652196226188text-muted style='opacity:0'>Generated on Oct. 1,
2023, 12:14 p.m..<o:p></o:p></p>

</body>

</html>

**Comment by nijel** [comment]
I see the error is no longer there. Did you figure out what was the cause?

**Reply by Mark-Giesen** [reply]
No I didn't, so I don't know what to do to prevent it or cure it next time. Because of "Noneype" in the error, I think of empty strings, but that's weird in YAML, there are keys without a value, but they are usually not a problem.

**Reply by nijel** [reply, marked as answer]
Unfortunately, we don't have more detailed info collected due to misconfiguration on our Sentry. But I'll watch for similar errors in future.

**Reply by Mark-Giesen** [reply]
OK thanks, I'll leave it for now then and we'll pick it up if it reoccurs.
